openid-4-verifiable-presentations-1_0.md

```diff
@@ -2249,6 +2247,45 @@ The VP Token contains the base64url-encoded `DeviceResponse` CBOR structure as d
 
 See ISO/IEC TS 18013-7 Annex B [@ISO.18013-7] and ISO/IEC 23220-4 Annex C [@ISO.23220-4] for the latest examples on how to use the `presentation_submission` parameter and how to generate the Authorizaton Response for presenting Credentials in the mdoc format.
 
+### `Handover` and `SessionTranscript` Definitions
+
+#### Invocation via the Digital Credentials API
+
+If the presentation request is invoked using the Digital Credentials API, the `SessionTranscript` CBOR structure as defined in Section 9.1.5.1 in [@ISO.18013-5] MUST be used with the following changes:
+
+* `DeviceEngagementBytes` MUST be `null`.
+* `EReaderKeyBytes` MUST be `null`.
+* `Handover` MUST be the `OID4VPDCAPIHandover` CBOR structure as defined below.
+
+```cddl
+OID4VPDCAPIHandover = [
+  "OID4VPDCAPIHandover", ; A fixed identifier for this handover type
+  OID4VPDCAPIHandoverInfoHash ; A cryptographic hash of OID4VPDCAPIHandoverInfo
+]
+OID4VPDCAPIHandoverInfoHash = bstr  ; sha-256 hash of OID4VPDCAPIHandoverInfo
+OID4VPDCAPIHandoverInfo = [
+  origin,
+  client_id,
+  nonce
+] ; Array containing handover parameters
+client_id = tstr  ; UTF-8 encoded string
+origin = tstr    ; UTF-8 encoded string
+nonce = tstr     ; UTF-8 encoded string
+```
+
+The `OID4VPDCAPIHandover` structure has the following elements:
+
+- The first element MUST be the fixed UTF-8 encoded string `"OID4VPDCAPIHandover"`. This serves as a unique identifier for the handover structure to prevent misinterpretation or confusion.
+- The second element MUST be the `OID4VPDCAPIHandoverInfoHash`, represented as a CBOR byte string which encodes the sha-256 hash of the `OID4VPDCAPIHandoverInfo` CBOR array.
+- The `OID4VPDCAPIHandoverInfo` has the following elements:
+  - The first element MUST be the UTF-8 encoded string representing the origin of the request as described in (#dc_api_request).
+  - The second element MUST be the UTF-8 encoded string value of the `client_id` request parameter.
```

Comment:
Shouldn't that be the effective Client Identifier (as defined in (#dc_api_request)? I'm asking since there is no `client_id` parameter in case of an unsigned DC API request.

Reply:
Yes, please review again. PR got shortened and previously had this info.